core/azure-core-amqp/pom.xml

```diff
@@ -39,9 +39,9 @@
     <url>https://github.com/Azure/azure-sdk-for-java</url>
     <connection>scm:git:https://github.com/Azure/azure-sdk-for-java.git</connection>
     <developerConnection>scm:git:https://github.com/Azure/azure-sdk-for-java.git</developerConnection>
-    <tag>v1.0.0-preview.1</tag>
+    <tag>v1.0.0-preview.3</tag>
```

Comment:
We should remove the tag as it doesn't add anything to duplicate the version here.